website/project/model.py

```diff
@@ -201,6 +223,10 @@ def edit(self, content, auth, save=False):
 
     def delete(self, auth, save=False):
         self.is_deleted = True
+        if self.page == 'files':
```

Comment:
Instead of using strings, can you make the valid values for page names named constants, e.g. `FILES = 'files'`

This has the benefit of preventing errors in comparison:

```python
self.page == 'file' // silent failure

FILES = 'files'
self.page == FILE  // loud failure (good)
```